src/main/java/io/vertx/core/eventbus/ClassNamePredicate.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2011-2022 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.eventbus;
+
+import io.vertx.codegen.annotations.VertxGen;
+
+import java.util.function.Predicate;
+
+/**
+ * A predicate invoked when verifying if an object can be encoded/decoded for transmission on the EventBus.
+ * <p>
+ * This is used by the system codecs for the subtypes of {@link io.vertx.core.shareddata.ClusterSerializable} and {@link java.io.Serializable}.
+ *
+ * @see EventBus#rejectSerializable(ClassNamePredicate)
+ * @see EventBus#allowSerializable(ClassNamePredicate)
+ * @see EventBus#rejectClusterSerializable(ClassNamePredicate)
+ * @see EventBus#allowClusterSerializable(ClassNamePredicate)
+ */
+@VertxGen
+@FunctionalInterface
+public interface ClassNamePredicate extends Predicate<String> {
+
+  /**
```

Comment:
You might as well put not docs here 😉

Reply:
Yeah, but that was really a simple oversight :smile: